jwst/stpipe/tests/test_step.py

```diff
@@ -592,14 +584,14 @@ def test_crds_override():
 
 
 def test_omit_ref_file():
-    from .steps import OptionalRefTypeStep
+    from jwst.stpipe.tests.steps import OptionalRefTypeStep
```

Comment:
Since you moved the other imports to the top of the module, might be worth moving this one too.